src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import duke.command.Command;
+
+/**
+ * Represents chatbot duke
+ */
+public class Duke {
+
+    private static final String filePath = "./data/duke.txt";
```

Comment:
Capitalising the variable name here would be in line with the coding standard for constants.